_includes/functions/matrix-cell-1-input.md

```diff
@@ -0,0 +1,37 @@
+{% capture /dev/null %}
+
+{% if include.state == "Full Support (send & receive)" %}
+  {% assign cell_emoji = "&#x2705;" %} <!--- Full Support --->
+  {% assign cell_tooltip = "Full Support (send & receive)" %} <!--- Full Support --->
+{% elsif include.state == "Send Support" %}
+  {% assign cell_emoji = "&#x1F4B8;" %}  <!--- Send Support --->
+  {% assign cell_tooltip = "Send Support" %} <!--- Send Support --->
+{% elsif include.state == "No Support" %}
+  {% assign cell_emoji = "&#x274C;" %}  <!--- No Support --->
+  {% assign cell_tooltip = "No Support" %} <!--- No Support --->
+{% elsif include.state == "Not Applicable" %}
+  {% assign cell_emoji = "&#x2796;" %} <!--- Not Applicable --->
+  {% assign cell_tooltip = "Not Applicable" %} <!--- Not Applicable --->
```

Comment:
Could you provide some guidance when to use "Not Applicable" vs "No Support"?

Reply:
I will refine this, as mentioned above.